plugins/cloud-gce/src/main/java/org/elasticsearch/cloud/gce/GceComputeServiceImpl.java

```diff
@@ -49,26 +48,26 @@
     implements GceComputeService {
 
     private final String project;
-    private final List<String> zoneList;
+    private final List<String> zones;
 
     // Forcing Google Token API URL as set in GCE SDK to
     //      http://metadata/computeMetadata/v1/instance/service-accounts/default/token
     // See https://developers.google.com/compute/docs/metadata#metadataserver
-    private static final String TOKEN_SERVER_ENCODED_URL = "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token";
+    public static final String TOKEN_SERVER_ENCODED_URL = "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token";
 
     @Override
     public Collection<Instance> instances() {
 
-            logger.debug("get instances for project [{}], zoneList [{}]", project, zoneList);
+            logger.debug("get instances for project [{}], zones [{}]", project, zones);
 
-            List<List<Instance>> instanceListByZone = Lists.transform(zoneList, new Function<String, List<Instance>>() {
+            List<List<Instance>> instanceListByZone = Lists.transform(zones, new Function<String, List<Instance>>() {
                 @Override
                 public List<Instance> apply(String zoneId) {
                     try {
                         Compute.Instances.List list = client().instances().list(project, zoneId);
                         InstanceList instanceList = list.execute();
                         if (instanceList.isEmpty()) {
-                            return new ArrayList();
+                            return Lists.newArrayList();
```

Comment:
I just saw @s1monw replacing ```Lists.newArrayList``` with ```new ArrayList<>()``` a few days ago. Maybe that way is out preferred way now?

+1 on removing the warning though.